odoo/tools/translate.py

```diff
@@ -670,7 +670,7 @@ def _process(format, modules, rows, buffer, lang):
             for module, type, name, res_id, src, trad, comments in rows:
                 row = grouped_rows.setdefault(src, {})
                 row.setdefault('modules', set()).add(module)
-                if not row.get('translation') and trad != src:
+                if not row.get('translation'):
```

Comment:
@mart-e 
This change is required because we need store the `row['translation']` even if they are the same